articles/cognitive-services/Speech-Service/releasenotes.md

```diff
@@ -15,6 +15,38 @@ ms.custom: seodec18
 
 # Speech Service release notes
 
```

Comment:
Look Level 5 8s not up for not selected or referenced persons, so sorry I don't  lack in knowing how these assigned area's work I don't have the kind set to block you but, inputs like these are annoying ab c,abc "a" is still over C5-H5 for time being so if you wanted more proper references about my actual assignment there I'm 8n process of contacting rental car for buy in and business owners after chkin with old service provider will make my self known, actual wanted customers,discount,information for guest involving specic location.thanks  much.jesneo.